fifa-crawler/fifa_parser/spiders/players_stats.py

```diff
@@ -52,6 +55,26 @@ def parse(self, response):
             player_tags = player.xpath('//div[@class="bp3-card double-spacing" and h5="Player Specialities"]//ul//li//a/text()').getall()
             player_traits = player.xpath('//div[@class="bp3-card double-spacing" and h5="Traits"]//ul//li//text()').getall()
 
+            # Add player club photo url and country photo url
+            player_country = player.xpath('//div[@class="meta bp3-text-overflow-ellipsis"]/a/@title').get()
+            player_country_logo_url = player.xpath('//div[@class="meta bp3-text-overflow-ellipsis"]//img/@data-src').get()
+            player_clubs_list = player.xpath('//div[@class="player-card double-spacing"]//h5/a/text()').getall()
+            player_logos_list = player.xpath('//div[@class="player-card double-spacing"]//img/@data-src').getall()
+            player_logos_urls = {}
+            country = {'name': player_country, 'url': player_country_logo_url}
+            player_logos_urls.update({'country': country})
+            for k, v in zip(player_clubs_list, player_logos_list):
+                if k == player_country:
+                    nationalClub = {'name': k, 'url': v}
+                    player_logos_urls.update({'nationalClub': nationalClub})
+                else:
+                    club = {'name': k, 'url': v}
+                    player_logos_urls.update({'club': club})
+
+            # Add player most played position
+            player_best_position = player.xpath('//div[@class="columns right"]/div[@class="column col-4"]//div[@class="double-spacing"]//span/text()').get()
```

Comment:
Can u do the renaming here too?

```suggestion
            player_primary_position = player.xpath('//div[@class="columns right"]/div[@class="column col-4"]//div[@class="double-spacing"]//span/text()').get()
```

And subsequently changing line136 too